document/CoreFormat.mdk

```diff
@@ -348,38 +352,38 @@ This shall be one of the following:
 *	`Tiger`
 *	`Whirlpool`
 
-## `issue` object { #issue }
+## `result` object { #result }
 
-### General { #issue-General }
+### General { #result-General }
 
-An `issue` object describes a single issue detected by a static analysis tool.
+A `result` object describes a single result produced by a static analysis tool.
```

Comment:
can we say something like a 'result' object describes a single diagnostic message (or simply message).
i.e., since we are defining 'result' here, let's define it using alternate terminology